src/screens/SelectionScreen/SelectionScreen.jsx

```diff
@@ -12,48 +14,69 @@ import vocascanCloud from "../../images/logo/vocascan-cloud.svg";
 
 import "./SelectionScreen.scss";
 
+import { Navigation, EffectCards } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/react/swiper-react";
+
 const SelectionScreen = () => {
   const { t } = useTranslation();
 
   const dispatch = useDispatch();
 
+  const isMdScreen = useMediaQuery("(max-width: 768px)");
+
   const handleSubmit = useCallback(
     (selfHosted) => {
       dispatch(setSelfHosted({ selfHosted }));
     },
     [dispatch]
   );
 
+  const selectionBoxes = [
+    <SelectionBox
+      key={0}
+      heading={t("screens.selectionScreen.vocascanCloud.heading")}
+      pro={t("screens.selectionScreen.vocascanCloud.pro", {
+        returnObjects: true,
+      })}
+      contra={t("screens.selectionScreen.vocascanCloud.contra", {
+        returnObjects: true,
+      })}
+      buttonText={t("screens.selectionScreen.vocascanCloud.button")}
+      image={vocascanCloud}
+      important
+      onSubmit={() => handleSubmit(false)}
+    />,
+    <SelectionBox
+      key={1}
+      heading={t("screens.selectionScreen.vocascanServer.heading")}
+      pro={t("screens.selectionScreen.vocascanServer.pro", {
+        returnObjects: true,
+      })}
+      contra={t("screens.selectionScreen.vocascanServer.contra", {
+        returnObjects: true,
+      })}
+      buttonText={t("screens.selectionScreen.vocascanServer.button")}
+      image={vocascanRound}
+      onSubmit={() => handleSubmit(true)}
+    />,
+  ];
+
+  if (isMdScreen) {
+    return (
+      <UnauthenticatedLayout>
```

Comment:
can we use the same layout for each screen and use the if in jsx?

Reply:
What do you mean particularly? Should I change every jsx file, that has the ternary-operator to if?

Reply:
No, I meant only return once and use the tenary op inside of jax to decide what needs to be rendered? Or do you think the current version is better? I mean if we change something, we need to change it on both returns.

Reply:
I prefer the ternary operator too. I can rebuild this